--- /etc/sysconfig/docker.ORIG	2021-03-19 16:50:31.000000000 +0300
+++ /etc/sysconfig/docker	2021-04-24 15:43:40.834779178 +0300
@@ -1,7 +1,7 @@
 # /etc/sysconfig/docker
 
 # Modify these options if you want to change the way the docker daemon runs
-OPTIONS='--selinux-enabled --log-driver=journald --signature-verification=false'
+OPTIONS='--log-driver=journald --signature-verification=false'
 if [ -z "${DOCKER_CERT_PATH}" ]; then
     DOCKER_CERT_PATH=/etc/docker
 fi
